fendermint/app/options/src/key.rs

```diff
@@ -48,6 +50,19 @@ pub struct KeyGenArgs {
     pub out_dir: PathBuf,
 }
 
+#[derive(Args, Debug)]
+pub struct EthPrivateToFendermintArgs {
+    /// Directory to the path that stores the private key
+    #[arg(long, short)]
+    pub path: PathBuf,
```

Comment:
```suggestion
    /// Path to the file that stores the private key (hex format)
    #[arg(long, short)]
    pub secret_key: PathBuf,
```

Similar to [TransArgs](https://github.com/consensus-shipyard/fendermint/blob/052505680f3b9791a6ee6ae8169973922442d7eb/fendermint/app/options/src/rpc.rs#L154-L156)